example.env

```diff
@@ -0,0 +1 @@
+REACT_APP_TMDB_API_KEY=
```

Comment:
Probably you wanted to say `.env.example` 🙂

Reply:
yos